book/en/src/by-example/app_init.md

```diff
@@ -3,30 +3,31 @@
 An RTIC application requires an `init` task setting up the system. The corresponding `init` function must have the
 signature `fn(init::Context) -> (Shared, Local)`, where `Shared` and `Local` are resource structures defined by the user.
```

Comment:
I'm just learning and not sure if true but I have a running example with a signature of:

```rust
    #[init]
    fn init(mut ctx: init::Context) -> (Shared, Local, init::Monotonics)
    ```
Is this paragraph outdated or incomplete?

Reply:
I see my example uses rtic v1 vs. rtic v2 here...